src/app.js

```diff
@@ -0,0 +1,69 @@
+import axios from "axios";
+
+const searchResult = document.getElementById('search-result');
+const searchForm = document.getElementById('search-form')
+searchForm.addEventListener('submit', searchCountry);
+const eMessage = document.getElementById('error-message');
+
+function searchCountry(e){
+e.preventDefault();
+    const searchValue = document.getElementById('search-value');
+    fetchData(searchValue.value);
+    searchValue.value = '';
+}
+async function fetchData(name){
+searchResult.innerHTML = ``;
+    try{
+const result = await axios.get(`https://restcountries.com/v2/name/${name}`)
+        const country = result.data[0];
```

Comment:
Je zou hier spreading kunnen toepassen, zodat je niet overal `country.` vóór hoeft te zetten:
```suggestion
        const { flag, region, name, subregion, population, capital, currencies, languages } = result.data[0];
```